### ISSUE TITLE
RFE: Custom config.php per Pico site

### ISSUE THREAD

**Issue by GregSutcliffe** [nextcloud/cms_pico#30]
Many of the custom themes available for Pico require you to add things to config.php for them to display properly. It's be great to be able to put a custom config.php in the Nextcloud directory to enable using such themes.

Example: https://github.com/xupefei/Travelify#installation

**Comment by daita** [comment]
adding config.php means that your users can execute PHP within the nextcloud environement, and it is a no go.

**Comment by GregSutcliffe** [comment]
There's no way to sandobx / sanitize it? Unfortunate. Presumably this means the only option is rewriting themes to use variables from the YAML frontmatter?

**Comment by daita** [comment]
citing @PhrozenByte YAML-based config are included in Pico 2.0. However, not sure he will find the time to backport it to 1.x.

**Comment by GregSutcliffe** [comment]
Yeah, I wouldn't expect that to happen - perhaps this ticket should become "Can the Nextcloud app be upgraded to Pico 2.0?" :)

**Comment by fbclol** [comment]
it's a shame that we can not change the config theme.
it breaks all the interest to have the possibility to have the choice between the themes and to have more sites.

I think that by thinking about it we could at least load a config by theme without being able to change it from the interface of nextcloud

**Comment by nsrosenqvist** [comment]
Why not just restrict it to only allow a config.yml in the root of the site? That way it must be parsed through YAML and won't allow arbitrary PHP code to run?
